.github/workflows/ci-shortfin-developer-tool-tests.yml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Shortfin Developer Tools Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run weekly to ensure our tools remain functional
+    - cron: '0 0 * * 0'
+  pull_request:
+    paths:
+      - '.github/workflows/ci-shortfin-developer-tool-tests.yml'
+      - 'app_tests/integration_tests/llm/shortfin/tracing_test.py'
+      - 'shortfin/**'
```

Comment:
Do we need a new workflow for this? Can this go in an existing unit test or integration test workflow instead?